src/main/java/com/github/onsdigital/babbage/search/external/requests/base/AbstractSearchRequest.java

```diff
@@ -79,7 +81,12 @@ public String getContentResponseAsString() throws Exception {
 
     @Override
     public T call() throws Exception {
-        String response = this.getContentResponseAsString();
+        ContentResponse contentResponse = this.getContentResponse();
+
+        if (contentResponse.getStatus() != HttpStatus.SC_OK) {
```

Comment:
Nit Picky comment _BUT_ in the interest of safety it might be worth adding a `if (contentResponse == null) ` check before the status check. I know it shouldn't ever happen but if it does you have explicitly checked and handled it appropriately.